modules/ossm-federation-config-smcp.adoc

```diff
@@ -0,0 +1,397 @@
+  cluster:
+    name:
+|Name of the cluster.  You are not required to specify a cluster name, but it is helpful for troubleshooting.
+|String
+|N/A
+
+|spec:
+  cluster:
+    network:
+|Name of the cluster network.  You are not required to specify a name for the network, but it is helpful for configuration and troubleshooting.
+|String
+|N/A
+|===
+
+== Understanding federation gateways
+
+You use a *gateway* to manage inbound and outbound traffic for your mesh, letting you specify which traffic you want to enter or leave the mesh.
+
+You use ingress and egress gateways to manage traffic entering and leaving the service mesh (North-South traffic).  When you create a federated mesh, you create additional ingress/egress gateways, to facilitate service discovery between federated meshes, communication between federated meshes, and to manage traffic flow between service meshes (East-West traffic).
+
+To avoid naming conflicts between meshes, you must create separate egress and ingress gateways for each mesh.  For example, 'red-mesh' would have separate egress gateways for traffic going to 'green-mesh' and `blue-mesh`.
+
+.Federation gateway parameters
+[options="header"]
+[cols="l, a, a, a"]
+|===
+|Parameter |Description |Values |Default value
+|spec:
+  gateways:
+    additionalEgress:
+      <egressName>:
+|Define an additional egress gateway for _each_ mesh peer in the federation.
+|
+|
+
+|spec:
+  gateways:
+    additionalEgress:
+      <egressName>:
+        enabled:
+|This parameter enables or disables the federation egress.
+|`true`/`false`
+|`true`
+
+|spec:
+  gateways:
+    additionalEgress:
+      requestedNetworkView:
+|Networks associated with exported services.
+|Set to the value of `spec.cluster.network` in the SMCP for the mesh, otherwise use <ServiceMeshPeer-name>-network.  For example, if the `ServiceMeshPeer` resource for that mesh is named `west`, then the network would be named `west-network`.
+|
+
+|spec:
+  gateways:
+    additionalEgress:
+      <egressName>:
+        router mode:
+|
+|`sni-dnat`
+|
+
+|spec:
+  gateways:
+    additionalEgress:
+      <egressName>:
+        service:
+          metadata:
+            labels:
+              federation.maistra.io/proxy:
+|Specify a unique label for the gateway to prevent federated traffic from flowing through the cluster's default system gateways.
+|
+|
+
+|spec:
+  gateways:
+    additionalEgress:
+      <egressName>:
+        service:
+          ports:
+|Used to specify the `port:` and `name:` used for TLS and service discovery.  Federation traffic consists of raw encrypted TCP for service traffic.
+|Port `15443` is required for sending TLS service requests to other meshes in the federation. Port `8188` is required for sending service discovery requests to other meshes in the federation.
+|
+
+|spec:
+  gateways:
+    additionalIngress:
+|Define an additional ingress gateway gateway for _each_ mesh peer in the federation.
+|
+|
+
+|spec:
+  gateways:
+    additionalEgress:
```

Comment:
Hi Julie,

The parameter is called additionalEgress and the description mentions ingress.

Reply:
Thanks, good catch.  Copy/paste error!